apps/src/code-studio/components/pairing/StudentSelector.jsx

```diff
@@ -48,27 +70,44 @@ export default class StudentSelector extends React.Component {
       return <span>{i18n.noStudentsInSection()}</span>;
     }
 
-    const studentDivs = this.props.students.map(student => {
-      let className = 'student selectable';
-      if (this.state.selectedStudentIds.indexOf(student.id) !== -1) {
-        className = 'student selectable selected';
+    const exceededMaximum = this.state.selectedStudentIds.length >= 4;
```

Comment:
NIT (i.e. only change if you want to)
There's a shorthand folks use sometimes to set get the needed props and state immediately when entering the render function. i.e.
```
render() {
  const {students} = this.props;
  const {selectedStudentIds} = this.state;
  ...
  const studentBtns = students.map(student => {
  ...
```